sphinx/tutorial/cylc/runtime/runtime-configuration.rst

```diff
@@ -236,12 +236,13 @@ off and carries on as normal.
 
       .. spoiler:: Solution warning
 
-         You can check your answers by running the following commands.
+         You can check your answers by running the following commands
+         (hyphens and colons optional):
```

Comment:
Consider adding a reminder here that they are optional **but can't be mixed**, given this seems to crop up a fair number of times! There is a warning to this effect in the 'Date-Time Cycling' section, but this would reinforce it at a later stage.